packages/components/src/List/ListComposition/DisplayMode/DisplayModeToggle.scss

```diff
@@ -3,11 +3,11 @@ $tc-display-button-icon-height: 2.4rem;
 .tc-display-mode-toggle {
 	display: flex;
 	// These divs element are the different action
-	div {
+	> * {
```

Comment:
this selector is too generic, is there any chance to put CSS class on them instead?